app/javascript/components/DualPaneMapper/DualPaneMapperCount.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DualPaneMapperCount = ({ selectedItems, totalItems }) => (
+  <div className="dual-pane-mapper-count">
+    {__(sprintf('%s of %s items selected', selectedItems, totalItems))}
```

Comment:
Thanks!
Just realized that it should have been  -
`sprintf(__('%s of %s items selected'), selectedItems, totalItems)`
(my apologies)

Otherwise the string would not be extracted for translation.